FAQ.md

```diff
@@ -129,13 +129,18 @@ Example scrape config:
 
 ```yaml
 scrape_configs:
-  - job_name: 'mbin-rabbit_queues'
+  - job_name: "mbin-rabbit_queues"
     static_configs:
-      - targets: ['example.org']
-    metrics_path: '/metrics/detailed'
+      - targets: ["example.org"]
+    metrics_path: "/metrics/detailed"
     params:
-      vhost: ['/']
-      family: ['queue_coarse_metrics', 'queue_consumer_count', 'channel_queue_metrics']
+      vhost: ["/"]
+      family:
+        [
+          "queue_coarse_metrics",
+          "queue_consumer_count",
+          "channel_queue_metrics",
+        ]
```

Comment:
Is this actually working or was it just the auto formatter?

Reply:
ow sorry, this is my editor automatically format markdown correctly.